src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -242,7 +242,32 @@ public boolean runUntilEnd() throws IOException, InterruptedException {
         ProcessRunner install = new ProcessRunner(sdkPath.resolve("platform-tools").resolve("adb").toString(),
                 "install", "-r", alignedApk);
         processResult = install.runProcess("install");
+        if (processResult != 0) throw new IOException("Application instalation failed!");
+
+        Runnable logcat = () -> {
+            try {
+                ProcessRunner clearLog = new ProcessRunner(sdkPath.resolve("platform-tools").resolve("adb").toString(),
+                "logcat", "-c");
+                clearLog.runProcess("clearLog");
+
+                ProcessRunner log = new ProcessRunner(sdkPath.resolve("platform-tools").resolve("adb").toString(),
+                "-d", "logcat", "-v", "brief", "-v", "color", "GraalCompiled:V", "GraalGluon:V", "AndroidRuntime:E", "ActivityManager:W", "*:S");
```

Comment:
this is a nice feature, but I wonder if we should have an option to toggle it on/off (the whole logcat thread). For example, I typically use logcat -v threadtime but depending on what you are doing, you want to try different options.

Reply:
`threadtime` would display the date and invocation time which are already added by ProcessRunner. I was aiming for most iOS-like option without losing vital information (since Android has no per app filter, several different tags needed to be included in order to catch crash logs). [Here](https://stackoverflow.com/a/48479579) is the nice breakdown of `logcat` format options, so we can decide whether there is more suitable option.

About toggling whole thread I guess we could use `verbose` configuration option, however I didn't see it used for runtime options yet, and `CTRL+C` would kill the `logcat` thread anyway.

Reply:
I was actually referring to make the whole log process toggled on/off.
As for the treadtime option: thread shows the process + parent process on android, and ProcessRunner doesn't have that. The time from the ProcessRunner is the received time, which can be different from the "android time".

Combining these remarks: the options here are perfect, and they fit well with the ios style. 

Thinking about it for a bit longer: you're totally right this should not even be an option. Android is the only platform where "client:run" didn't result in a run, so it has to be changed to be in line with the others. We are thinking about other lifecycle events (e.g. install, package), but that is another issue.

Reply:
I guess this option will be the default one you get with `client:run`, but you can always go to a terminal and run `adb` with other options if needed.